gitlab_users

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+from lib import get_gitlab_instance
+
+import sys
+
+SHOW_CONFIG = len(sys.argv) >= 2 and sys.argv[1] == 'config'
+
+if SHOW_CONFIG:
+    print('graph_title GitLab users')
+    print('graph_vlabel users')
+    print('graph_args -l 0')
+    print('graph_category gitlab')
```

Comment:
did you read my comment? 
i do not return values, i return the fields munin needs to draw. that's how munin works.